releases/release-1.22/release-notes/maps/pr-99647-map.yaml

```diff
@@ -0,0 +1,11 @@
+pr: 99647
+releasenote:
+  text: '`kubeadm`: remove the automatic detection and matching of cgroup drivers
```

Comment:
Hyper link should shorten this bit a lot and also on a few places the "" are used instead ``